src/components/Header/index.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import AddIcon from '@mui/icons-material/Add';
+import LanguageIcon from '@mui/icons-material/Language';
+import MenuIcon from '@mui/icons-material/Menu';
+import SearchIcon from '@mui/icons-material/Search';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import logo from 'assets/img/logo.svg';
+
+interface IPage {
+	label: string;
+	pages: string[];
+}
+
+export const Header = () => {
+	const navItems: IPage[] = [
+		{ label: 'Movies', pages: ['Popular', 'Now playing', 'Upcoming', 'Top rated'] },
```

Comment:
{label: string, value: {title: string, path: string (the path need for a react router dom)